### PR TITLE
Add a continuous load-test script.

### PR DESCRIPTION
**Pull request by tylerwilliams**

### CODE REVIEW

File: tools/cacheload/cacheload.go [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+		}
+		low = histBuckets[i+1]
+	}
+	return randRange(histBuckets[len(histBuckets)-2], histBuckets[len(histBuckets)-1])
+}
+
+var generatorMu = sync.Mutex{}
+
+func newRandomDigestBuf(sizeBytes int64) (*repb.Digest, []byte) {
+	generatorMu.Lock()
```
Comment (vadimberezniker) [inline tools/cacheload/cacheload.go:99]: can you put this lock inside the Generator?

File: tools/cacheload/cacheload.go [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+	defer generatorMu.Unlock()
+	d, buf, err := digestGenerator.RandomDigestBuf(sizeBytes)
+	if err != nil {
+		log.Fatalf("Error generating digest: %s", err)
+	}
+	return d, buf
+}
+
+func writeBlob(ctx context.Context, client bspb.ByteStreamClient) (*repb.Digest, error) {
```
Comment (vadimberezniker) [inline tools/cacheload/cacheload.go:108]: Can you update writeBlob and readBlob to wrap the returned error to include the digest being read/written?

**Review by vadimberezniker — APPROVED** [review]

File: tools/cacheload/cacheload.go (hunk 1 above)
Comment (tylerwilliams) [reply tools/cacheload/cacheload.go]: Done.

File: tools/cacheload/cacheload.go (hunk 2 above)
Comment (tylerwilliams) [reply tools/cacheload/cacheload.go:105]: Done.

**Pull request merged**